SimCalorimetry/CastorSim/plugins/CastorDigiAnalyzer.h

```diff
@@ -9,8 +9,8 @@
 #include "FWCore/Utilities/interface/InputTag.h"
```

Comment:
This file was included in CastorDigiAnalyzer.cc, and it should therefore get removed